WordPress/Classes/Utility/ViewLoading/NibLoadable.swift

```diff
@@ -20,7 +20,7 @@ public extension NibLoadable {
     }
 
     static var defaultBundle: Bundle {
-        return Bundle(for: Self.self)
+        return Bundle.keystone
```

Comment:
What's the advantage of using `Bundle.keystone` over `Bundle(for: Self.self)`?

Personally, I like `Bundle(for: Self.self)` because if this util goes into another framework, it will continue to work.

Reply:
One thing I'll say in favor of `Bundle.keystone` is that it is a consistent approach across `struct` and `class`, which is nice.

If we used `Bundle(for: Self.self)`, it would work on all `class`es, but we'd need an ad hoc solution for `struct`s...

<img width="450" alt="image" src="https://github.com/user-attachments/assets/bbcfaf91-c890-46d2-8054-9c9af293ecd2" />